modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -1471,6 +1588,9 @@ Status description:: error: procedure exception - too many seeders. The number o
 
 Status description:: error: procedure exception - no such seeder. The specified seeding server with id `{ $server }` was not found. Verify that the spelling is correct.
 
+=== 52N22
+
+Status description:: error: procedure exception - invalid procedure argument. Invalid argument `{ $value }` for `{ $procParam }` on procedure `{ $proc }`. The expected format of `{ $procParam }` is `{ $procParamFmt }`.
```

Comment:
```suggestion
Status description:: error: procedure exception - invalid procedure argument. Invalid argument `{ $field }` for `{ $procParam }` on procedure `{ $proc }`. The expected format of `{ $procParam }` is `{ $procParamFmt }`.
```

This one was inconsistent between spreadsheet and codebase, I have updated the spreadsheet now